### PR TITLE
Remove chromedriver as a dependency

### PR DESCRIPTION
**Pull request by iksinski**
PR will remove chromedriver as a dependency, using parent repo chromedriver instead. Thanks to that we're not bound to the version of chrome.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,7 +83,6 @@
     "chai-as-promised": "^6.0.0",
     "child-process-debug": "0.0.7",
     "chokidar": "^2.1.6",
-    "chromedriver": "^2.46.0",
```
Comment (lgandecki) [inline package.json]: we should add this as peerDependency if it's a required dependency of the parent project

File: src/bin/default.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
 import path from 'path';
 import {isCI} from '../lib/ci';
 
+const chromedriver = require('../../../../package').devDependencies.chromedriver;
+
+if (!chromedriver) {
+  throw new Error('Please install chromedriver as a dev dependency in your repo!')
```
Comment (lgandecki) [inline src/bin/default.js]: not repo, project/package. Would be good to add instructions or link about how to do so.

File: src/bin/default.js [hunk 3]
```diff
@@ -1,6 +1,12 @@
 import path from 'path';
 import {isCI} from '../lib/ci';
 
+const chromedriver = require('../../../../package').devDependencies.chromedriver;
```
Comment (lgandecki) [inline src/bin/default.js]: that seems problematic, maybe something like https://www.npmjs.com/package/find-package-json . It uses cwd() by default so it should find a proper package.json in most cases.

File: src/lib/chromedriver.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-var chromedriver = require('chromedriver'),
+var chromedriver = module.parent.require('chromedriver'),
```
Comment (lgandecki) [inline src/lib/chromedriver.js]: I think this means we break tests in the chimpy itself. Maybe we should have the chromedriver as both peerDependency and devDependnecy, so when installing in other projects it doesn't come with it's own, but when installed on its own it does have a chromedriver to use

**Comment by danrice92** [comment]
Can someone please merge a fix for this?

**Comment by rsshilli** [comment]
This is great!  Thank you!

**Pull request merged**